src/android/IndoorAtlasWayfinding/IAWayfinding.java

```diff
@@ -0,0 +1,127 @@
+package com.iawayfinding.plugin;
+
+import org.apache.cordova.CordovaPlugin;
+import org.apache.cordova.CallbackContext;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import java.util.ArrayList;
+
+import com.indooratlas.android.wayfinding.IARoutingLeg;
+import com.indooratlas.android.wayfinding.IARoutingPoint;
+import com.indooratlas.android.wayfinding.IAWayfinder;
+
+/**
+ * IAWayfinding wrapper for Android
+ */
+public class IAWayfinding extends CordovaPlugin {
```

Comment:
We should not need a separate Cordova plugin for wayfinding anymore, just put the initWithGraph and computeRoute in the main plugin (?)